src/sync/multichain/multichain.d.ts

```diff
@@ -10,6 +10,8 @@ declare module 'multichain' {
   export interface Connection {
     getAddresses(): Promise<string[]>
 
+    dumpPrivKey(address): Promise<string>
```

Comment:
moet deze niet `getPrivateKey` zijn? (wat ik ook mooier vinden dan 'dump' 😉 )

Reply:
Het heet dumpPrivKey in de multichain RPC-api, kan ik helaas niet aanpassen. In onze methodes is het wel getPrivateKey :)